### PR TITLE
Bug 1222215 - Stop using per-file retry scheme. r=mhenretty

### PR DESCRIPTION
**Pull request by nullaus**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=80a5920fbf8c49400f457501cf80b81fd30468de

File: bin/gaia-marionette/gaia-marionette [hunk 1]
```diff
@@ -176,12 +176,7 @@ else
   DEVICE_TYPE_OPT="--device-type $GAIA_DEVICE_TYPE"
 fi
 
-TARGET=""
-if [[ $@ == *"mocha-tbpl-reporter"* ]]; then
-  TARGET="$DIR/../gaia-marionette/gaia_marionette_retry.js"
```
Comment (gaye) [inline bin/gaia-marionette/gaia-marionette]: Maybe also delete that file?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=735dd67cdb565f152e86f43506836a5f6e134b71

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=670932ce0cee69cb13d62e9a11d9ea54a9d69233

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eee7e3801553309070f7f15037d190ef89fd2d5b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8b330e56a6dece8ae77fb9ffc86949fd18755c07

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f321d8a02718a182fd18f61ea8838c7459f8ae8b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=828d17a2b1af3732ec866e5ffbcb04ce42a6a14a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=828d17a2b1af3732ec866e5ffbcb04ce42a6a14a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0047678413f9491af530335279476a0139a616a0

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=30f6ea2a965cdbcfcb9c5ddd5ca9773fb3961a48

**Pull request merged**